04-class-field.tex

```diff
@@ -60,7 +60,7 @@ \section{Property}
 
 \haxe{assets/Property.hx}
 
-For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by
+For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by:
```

Comment:
No, I am not sure at all. I was just matching the style bellow, it makes 
total sense without it.

On 05-01-2015 19:03, Simon Krajewski wrote:
>
> In 04-class-field.tex 
> <https://github.com/HaxeFoundation/HaxeManual/pull/79#discussion-diff-22489336>:
>
> > @@ -60,7 +60,7 @@ \section{Property}
> >
> >  \haxe{assets/Property.hx}
> >
> > -For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by
> > +For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by:
>
> Are you sure about that colon? I'm pretty sure I use this style 
> (sentences which have listing in between) in several places. I'm also 
> pretty sure I've seen it in other sources.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/HaxeFoundation/HaxeManual/pull/79/files#r22489336>.
>

-- 
Arthur
arthur.szasz@gmail.com
(11) 9 6309 - 0246
(11) 3895 - 6753